clang/tools/clang-sycl-linker/ClangSYCLLinker.cpp

```diff
@@ -338,6 +382,87 @@ static Error runSPIRVCodeGen(StringRef File, const ArgList &Args,
   return Error::success();
 }
 
+/// Run AOT compilation for Intel CPU.
+/// Calls opencl-aot tool to generate device code for Intel CPU backend.
```

Comment:
```suggestion
/// Calls opencl-aot tool to generate device code for Intel CPU.
```
What is the meaning of "Intel CPU **backend**"?
The applies to GPU.

Reply:
Changed to `Calls opencl-aot tool to generate device code for the Intel OpenCL CPU Runtime` / `Calls ocloc tool to generate device code for the Intel Graphics Compute Runtime`.